src/content/iconsSheet.svg

```diff
@@ -312,6 +312,7 @@
 <symbol id="icon-grid-view" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 21 21">
  <path stroke="none" fill-rule="evenodd" d="M18.5,17.5 L18.5,15.5 L16.5,15.5 L16.5,17.5 L18.5,17.5 Z M19,18.5 L16,18.5 C15.7,18.5 15.5,18.3 15.5,18 L15.5,15 C15.5,14.7 15.7,14.5 16,14.5 L19,14.5 C19.3,14.5 19.5,14.7 19.5,15 L19.5,18 C19.5,18.3 19.3,18.5 19,18.5 Z M4.5,5.5 L4.5,3.5 L2.5,3.5 L2.5,5.5 L4.5,5.5 Z M5,6.5 L2,6.5 C1.7,6.5 1.5,6.3 1.5,6 L1.5,3 C1.5,2.7 1.7,2.5 2,2.5 L5,2.5 C5.3,2.5 5.5,2.7 5.5,3 L5.5,6 C5.5,6.3 5.3,6.5 5,6.5 Z M4.5,11.5 L4.5,9.5 L2.5,9.5 L2.5,11.5 L4.5,11.5 Z M5,12.5 L2,12.5 C1.7,12.5 1.5,12.3 1.5,12 L1.5,9 C1.5,8.7 1.7,8.5 2,8.5 L5,8.5 C5.3,8.5 5.5,8.7 5.5,9 L5.5,12 C5.5,12.3 5.3,12.5 5,12.5 Z M4.5,17.5 L4.5,15.5 L2.5,15.5 L2.5,17.5 L4.5,17.5 Z M5,18.5 L2,18.5 C1.7,18.5 1.5,18.3 1.5,18 L1.5,15 C1.5,14.7 1.7,14.5 2,14.5 L5,14.5 C5.3,14.5 5.5,14.7 5.5,15 L5.5,18 C5.5,18.3 5.3,18.5 5,18.5 Z M11.5,5.5 L11.5,3.5 L9.5,3.5 L9.5,5.5 L11.5,5.5 Z M12,6.5 L9,6.5 C8.7,6.5 8.5,6.3 8.5,6 L8.5,3 C8.5,2.7 8.7,2.5 9,2.5 L12,2.5 C12.3,2.5 12.5,2.7 12.5,3 L12.5,6 C12.5,6.3 12.3,6.5 12,6.5 Z M11.5,11.5 L11.5,9.5 L9.5,9.5 L9.5,11.5 L11.5,11.5 Z M12,12.5 L9,12.5 C8.7,12.5 8.5,12.3 8.5,12 L8.5,9 C8.5,8.7 8.7,8.5 9,8.5 L12,8.5 C12.3,8.5 12.5,8.7 12.5,9 L12.5,12 C12.5,12.3 12.3,12.5 12,12.5 Z M11.5,17.5 L11.5,15.5 L9.5,15.5 L9.5,17.5 L11.5,17.5 Z M12,18.5 L9,18.5 C8.7,18.5 8.5,18.3 8.5,18 L8.5,15 C8.5,14.7 8.7,14.5 9,14.5 L12,14.5 C12.3,14.5 12.5,14.7 12.5,15 L12.5,18 C12.5,18.3 12.3,18.5 12,18.5 Z M18.5,5.5 L18.5,3.5 L16.5,3.5 L16.5,5.5 L18.5,5.5 Z M19,6.5 L16,6.5 C15.7,6.5 15.5,6.3 15.5,6 L15.5,3 C15.5,2.7 15.7,2.5 16,2.5 L19,2.5 C19.3,2.5 19.5,2.7 19.5,3 L19.5,6 C19.5,6.3 19.3,6.5 19,6.5 Z M18.5,11.5 L18.5,9.5 L16.5,9.5 L16.5,11.5 L18.5,11.5 Z M19,12.5 L16,12.5 C15.7,12.5 15.5,12.3 15.5,12 L15.5,9 C15.5,8.7 15.7,8.5 16,8.5 L19,8.5 C19.3,8.5 19.5,8.7 19.5,9 L19.5,12 C19.5,12.3 19.3,12.5 19,12.5 Z"/>
 </symbol>
+<symbol id="icon-html-templates" /></symbol>
```

Comment:
Am I missing something here? Shouldn't there be a path?

Reply:
that is created by the script ... this is weird ...